src/components/layout/nav.js

```diff
@@ -619,17 +619,46 @@ const HomeContainer = styled(Container)`
 
 const StyledNavCenter = styled(NavCenter)`
     margin-left: 13.3rem;
+    white-space: nowrap;
 
     @media (max-width: 1300px) {
-        margin-left: 7.3rem;
+        margin-left: 9.3rem;
+        font-size: 12px !important;
     }
     @media (max-width: 1080px) {
-        margin-left: 2.4rem;
+        margin-left: 7.3rem;
     }
 `
 
 const StyledNavRight = styled(NavRight)`
```

Comment:
add styling as well for the languageswitcher inside navright
currently on `chinese language`, the text is broken